rcp/org.testeditor.rcp4.uatests/src/main/java/org/testeditor/fixture/swt/TestEditorFixture.java

```diff
@@ -106,11 +106,11 @@ public String isValidProject(String project) throws CoreException {
 	}
 
 	@FixtureMethod
-	public String containsWorkspaceFileText(String filePath, String searchText) throws CoreException, IOException {
+	public boolean containsWorkspaceFileText(String filePath, String searchText) throws CoreException, IOException {
 		logger.info("Cearching for text {} in {}", searchText, filePath);
 		IWorkspaceRoot root = ResourcesPlugin.getWorkspace().getRoot();
-		return Boolean.toString(Files.lines(new File(root.getLocation().toString() + filePath).toPath())
-				.filter(s -> s.contains(searchText)).count() > 0);
+		return Files.lines(new File(root.getLocation().toString() + filePath).toPath())
```

Comment:
We should move the `new File` into a separate line to keep this shorter. Also use `new File(parent, child)`